src/components/Main/Header.tsx

```diff
@@ -0,0 +1,58 @@
+import { NavLink } from 'react-router-dom';
+
+import styled from 'styled-components';
+
+const Container = styled.div`
+    max-width: 600px;
+    height: 69px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: .7em;
+    margin-bottom: .7em;
```

Comment:
취향이라고 볼 수도 있긴 한데 보통 단축해서 많이 쓰는 것 같아요~
```suggestion
    margin: .7em 0;
```